tools/runners/run-test-suite.sh

```diff
@@ -154,7 +158,8 @@ do
 
         if [ $status_code -eq 0 ]
         then
-            test $VERBOSE && echo "[$tested/$TOTAL] $cmd_line: PASS"
+            test $VERBOSE && printf "[%4d/%4d] " "$tested" "$TOTAL" &&
+                echo " PASS: $cmd_line"
```

Comment:
What is the reason for this line break? And why is this echo not merged into the printf after all?

Reply:
You're right, I will merge these two commands into one.